src/gui/entry/EditEntryWidget.cpp

```diff
@@ -28,6 +29,7 @@
 #include <QMenu>
 #include <QSortFilterProxyModel>
 #include <QTemporaryFile>
+#include <QtWidgets>
```

Comment:
Remove this include line to prevent compile errors on Windows.

Reply:
I think removing it will make Travis fail with older Qt 5.

Reply:
Perhaps you are missing a more specific include then?

Reply:
One of you told me on IRC to include QtWidgets instead of the specific ones when I had them in the early incarnations. Please decide.

Reply:
Personally, I don't care. But if including QWidgets produces errors on Windows, this needs to be addressed.

Reply:
That was throw away advice to get you going. You always want to scope down if possible, and especially if it causes issues. The ui_XXXX.h files bring in all the widget headers you need. It was likely a false positive that Travis was barfing on a widget file.